ci/travis/ci.sh

```diff
@@ -452,6 +423,7 @@ init() {
 
 build() {
   if ! need_wheels; then
+    bazel_ensure_buildable
```

Comment:
this is ran both in need_wheels and !need_wheels, can we change it to `bazel_ensure_buidable_on_windows` or something and run it out side of the if condition

Reply:
Done